.github/workflows/create-release.yml

```
@@ -0,0 +1,34 @@
+name: Create Release (Versioned)
+
+on:
+  workflow_run:
+      workflows: ["Publish NPM Package"]
+      types:
+        - completed
+
+jobs:
+  create-release:
+    # only run if the "Publish NPM Package" workflow was successful
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Node
+      uses: actions/setup-node@v3
+    - name: Git Fetch       # Necessary for the changelog to generate everything as intended
+      run: git fetch --prune --unshallow --tags
+    - name: Set Current Package Version as a Variable
+      id: package-version   # Can be referenced with ${{ steps.package-version.outputs.current-version }}
+      uses: martinbeentjes/npm-get-version-action@main
+    - name: Create Changelog Config
+      # For more info on the auto-changelog config params: https://www.npmjs.com/package/auto-changelog/v/2.4.0
+      run: echo '{"output":"CHANGELOG.md","template":"keepachangelog","package":true,"unreleased":false,"commitLimit":false,"sortCommits":"date-desc"}' > .auto-changelog
```

@nick-w-nick Do you think we want it to include links to any relevant GitHub issues in this repo? If so, I think we can achieve that like so. Double check my `issueUrl` and especially `issuePattern`. I think they should be right, though it wouldn't hurt to double check me. I think atm, this would make any number link to the relevant issue with the `issuePattern`, and I'm not sure if we want that.
```suggestion
      # For more info on the auto-changelog config params: https://www.npmjs.com/package/auto-changelog/v/2.4.0
      run: echo '{"output":"CHANGELOG.md","template":"keepachangelog","package":true,"unreleased":false,"commitLimit":false,"sortCommits":"date-desc","issueUrl":"https://github.com/nick-w-nick/eslint-config-lintification/issues/{id}","issuePattern":"\\b(\\d+)\\b}' > .auto-changelog
```

---

Great idea, that pattern looks like it should work based on my testing. Committed!